src/LevelDBStore/IO/Data/LevelDB/Helper.cs

```diff
@@ -7,19 +7,46 @@ namespace Neo.IO.Data.LevelDB
 {
     public static class Helper
     {
+        private static int ByteArrayCompare(byte[] x, byte[] y)
```

Comment:
Since LevelDBStore project is netstandard2.1, you can use Span<T>.SequenceCompare instead of having to write your own ByteArrayCompare function